grouper/fe/handlers/search.py

```diff
@@ -25,13 +27,16 @@ def get(self):
             .subquery()
         )
 
+        permission_query = query.split("=")[0] if "=" in query else query
         permissions = (
             self.session.query(
                 label("type", literal("Permission")),
                 label("id", Permission.id),
                 label("name", Permission.name),
             )
-            .filter(Permission.enabled == True, Permission.name.like("%{}%".format(query)))
+            .filter(
+                Permission.enabled == True, Permission.name.like("%{}%".format(permission_query))
```

Comment:
Note that because the argumented search only triggers when there's only a single result, you won't be able to do an argumented search for any permission whose name is contained in another permission or group name. For example, if there are permissions `foo` and `foo.bar`, trying to search for `foo=baz` will turn up both permissions and thus will not have the special argumented search behavior.

Honestly I think it's fine, the feature is still an improvement and it'll be possible to manually tack on `?permission_arg=baz` on the permission view page. Alternatively could change the behavior by making the search be for an exact match if there is a `=`, but that behavior doesn't deem ideal either